src/webots/nodes/WbRotationalMotor.cpp

```diff
@@ -84,6 +85,10 @@ void WbRotationalMotor::turnOffMotor() {
 }
 
 double WbRotationalMotor::computeFeedback() const {
+  const WbPropeller *propeller = dynamic_cast<WbPropeller *>(parentNode());
+  if (propeller)
+    return fabs(propeller->currentThrust()) + fabs(propeller->currentTorque());
```

Comment:
Is that correct? Can we simply add a force to a torque? That seems odd... In the case of a rotational motor, the feedback is supposed to be a torque. So the thrust should be somehow converted to the equivalent torque.

Reply:
This is indeed an approximation, but we anyway claim in the documentation that the consumption of motors is a rough estimation. We define it as `force * consumptionFactor` for motors generating a force (LinearMotor) or as `torque * consumptionFactor` for motors generating a torque (RotationalMotor). In the case of Propeller, it is special, it is a RotationalMotor, but it is generating both a torque and a force (thrust), so the idea here is to take both into account for the consumption, for converting the force (thrust) to a moment we are here missing information (we need a kind of 'Lever Arm Length', we could introduce a new field to the propeller, but this looks overkilled to me, especially because we are then using an extremely approximated model to convert torque to consumption).
Otherwise, another solution would be to do like we did for Tracks (simply disabling consumption for motor in Propellers).

Reply:
I am not sure we need a 'lever arm length' as it could probably be computed (or estimated) from the torque and thrust constants. But I am not sure about the formula... So, let's proceed with this approximation, although it looks wrong... Maybe some comment in the code would be helpful, so that we know it's wrong.

Reply:
Intuitively, I would say it should be something like torque feedback = currentCurrentTorque + currrentForce * (q1/t1 + q2/t2).

Reply:
This formula looks indeed intuitively rights. But there are some cases which don't make sense e.g.:
Imagine a 'perfect' propeller (produce thrust without resistant torque), then q1 and q2 and currentTorque = 0, but currentForce != 0, this means we return a feedback of 0, but some thrust is produced.
And the problem is even worse if t1 or t2 is 0 (which is the case with the default fields value).

Reply:
OK, so let write a comment in there until we find a better formula.